src/main.css

```diff
@@ -8,9 +8,8 @@
 }
 
 html {
-    background: #00162d;
-    box-sizing: border-box;
```

Comment:
Why remove this?

Reply:
I will add the box-sizing back, didn't realise I deleted that. My bad